index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+</head>
+<body>
+
+    
+         <header>
+            <a id="logo">My Logo</a><br><br>
+        <nav class="manu">
+            <img id="img2" src="https://icon-library.com/images/menu-icon-png-3-lines/menu-icon-png-3-lines-10.jpg" alt="#" class="menu">
+    </nav>
+        </header>
+    <section class="headline">
+    <div class="flex-container">
+        <h1>Hey There I'm Alex</h1>
+        <h2>I'am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </div>
+   </section>
+      
+<div class="bottom">
+    <div class="columns-wrapper">
+        <div class="columns">
+            <ul>
+<li><img id="img3" src="Vector.png" alt="#"></li>
+<li><img id="img4" src="Group.png" alt="#"></li>
+<li><img id="img5" src="1Vector.png" alt="#"></li>
+<li><img id="img6" src="2Vector.png" alt="#"></li>
```

Comment:
- [ ] [OPTIONAL] Thank you for following the previous reviewer's request to use `ul` and `li` 👏👏. sorry about this but what he forgot to mention is that inside the `li` tags should also be `a` tags. I've made this optional. but I would strongly recommend that you do this.
something like this should be really good
```suggestion
<li><a href="#"><img id="img3" src="Vector.png" alt="#"></a></li>
<li><a href="#"><img id="img4" src="Group.png" alt="#"></a></li>
<li><a href="#"><img id="img5" src="1Vector.png" alt="#"></a></li>
<li><a href="#"><img id="img6" src="2Vector.png" alt="#"></a></li>
```